### PR TITLE
Fix cluster/manager RBAC permissions guide

### LINKED ISSUE #6808: Add basic manager/cluster permissions to RBAC guide
Hi team, 
there are some basic permissions that Wazuh Dashboard needs to work properly and are not included in the documentation section:
https://documentation.wazuh.com/4.7/user-manual/user-administration/rbac.html

These permissions are related to reading the cluster/manager configuration. 

The needed permissions are:
| Action | Resources |
| --- | --- |
| manager:read  | `*:*:*` |
| manager:read_api_config  | `*:*:*` |
| cluster:status  | `*:*:*` |
| cluster:read  | `node:id:*` |
| cluster:read_api_config  | `node:id:*` |

We need to improve the RBAC guide to prevent permission errors in Wazuh Dashboard.

### PR DESCRIPTION
**Pull request by asteriscos**
## Description
Hi team,
this pull request adds the minimum cluster configuration reading role to the user role mapping setup.

Closes #6808 

### Docs building
- [ ] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by Desvelao — APPROVED** [review]
LGTM!

File: source/user-manual/user-administration/rbac.rst [hunk 1]
```diff
@@ -339,7 +339,7 @@ To map the user with Wazuh, follow these steps:
 #. Click **Create Role mapping** and complete the empty fields with the requested information.
    
       - **Role mapping name**: Assign a name to the role mapping.
-      - **Roles**: Select the role created previously. 
+      - **Roles**: Select the role created previously. Then select the **cluster_readonly** role to assign the user basic configuration reading permissions.
```
Comment (javimed) [inline source/user-manual/user-administration/rbac.rst:342]:
```suggestion
      - **Roles**: Select the role created previously and  the ``cluster_readonly`` role. This role assigns the user basic configuration reading permissions.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ All notable changes to this project will be documented in this file.
 
 - Support for Wazuh 4.7.2.
 
+### Changed
+
+- Updated the use case: Give a user permissions to read and manage a group of agents with minimum configuration reading permissions. ([#6761](https://github.com/wazuh/wazuh-documentation/pull/6761))
```
Comment (javimed) [inline CHANGELOG.md:12]: What use case? And the PR reference doesn't look correct

**Review by javimed — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (asteriscos) [reply CHANGELOG.md]: Good catch

**Review by javimed — APPROVED** [review]

**Pull request merged**